pyproject.toml

```diff
@@ -60,6 +60,7 @@ apache-airflow-client = "^2.9.0"
 beautifulsoup4 = "^4.12.3"
 django-allauth = {extras = ["socialaccount"], version = "^0.63.2"}
 django-login-required-middleware = "^0.9.0"
+django-s3-file-field = {extras = ["minio"], version = "^1.0.1"}
```

Comment:
The `minio` variant should only be installed in development. In production, we'll need the `s3` variant.

You can run these commands to get poetry to do it:

```bash
poetry remove django-s3-file-field
poetry add django-s3-file-field --extras "s3"
poetry add django-s3-file-field --group dev --extras "minio"
```